tools/azure-sdk-tools/packaging_tools/generate_utils.py

```diff
@@ -126,3 +127,31 @@ def judge_tag_preview(path: str) -> bool:
 
     _LOGGER.info(f'find single api version:{api_version}')
     return 'preview' in api_version
+
+
+def format_samples(sdk_code_path) -> None:
+    generate_sample_path = Path(sdk_code_path + '/generate_sample')
+    if not os.path.exists(generate_sample_path):
+        _LOGGER.info(f'not find generate_sample')
+        return
+
+    try:
+        import black
+    except Exception as e:
+        check_call('pip install black', shell=True)
+        import black
+
+    _BLACK_MODE = black.Mode()
+    _BLACK_MODE.line_length = 120
+    files = generate_sample_path.glob('**/*.py')
+    for path in files:
+        with open(path, 'r') as fr:
+            file_content = fr.read()
+
+        with suppress(black.NothingChanged):
```

Comment:
what is the usage of `supress`? Could we remove it?

Reply:
> what is the usage of `surpress`? Could we remove it?

It means to ignore the exception, which is more concise
```
with surpress(FooError): 
    foo()

# equal

try:
    foo()
except FooError:
    pass 
```